app/controllers/connections/gemini_connections_controller.rb

```diff
@@ -1,28 +1,29 @@
 # frozen_string_literal: true
 
-module Publishers
+module Connections
```

Comment:
IMO Connections is borderline a "safeword" since it's related to database connections. Would prefer `PaymentConnections` instead

Reply:
https://github.com/brave-intl/publishers/pull/2849/commits/782a65bafbca333dc83ec84a154ae5eaba3d7763